style.css

```diff
@@ -89,3 +139,705 @@ body {
+}
+
+.story-details {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  max-width: 327px;
+}
+
+.langs {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+  flex-wrap: wrap;
+}
+
+.recent-works h5 {
+  margin: 0;
+}
+
+.disp-btn {
+  display: flex;
+  justify-content: left;
+  width: 327px;
+  border: none;
+}
+
+.pst-btn {
+  background: #ff6b00;
+  margin: 12px 12px;
+  border: none;
+  height: 48px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  max-width: 100%;
+  padding: 5px;
+  text-align: center;
+  box-shadow: 0 0 5px -1px rgba(0, 0, 0, 0.2);
+}
+
+.projects {
+  align-self: flex-start;
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 8px;
+  font-family: Crete Round, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: 64px 0 0 12px;
+  padding-top: 12px;
+}
+
+a:any-link {
+  color: #fff;
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+.projects-details {
+  align-self: flex-start;
+  width: 295px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 12px;
+}
+
+.langs-2 {
+  display: flex;
+  justify-content: left;
+  width: 100%;
+  margin-left: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.work-2 {
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.d-work {
+  display: none;
+}
+
+.proj-work-j {
+  height: 386px;
+  width: 327px;
+  background-image: url(../img/Img-Placeholder-0.png);
+  background-repeat: no-repeat;
+  margin: 24px;
+  transition: 1.5s ease-out;
+}
+
+.proj-work-j:hover {
+  transform: scale(1.1);
+}
+
+.proj-btn {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  border: none;
+  height: 48px;
+  background: #ff6b00;
+  margin: 12px 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.see {
+  border: none;
+  background: #ff6b00;
+  padding: 12px 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  width: 100%;
+}
+
+.modal-footer {
+  display: flex;
+  justify-content: space-around;
+  margin: 0;
+}
+
+.about {
+  margin-top: 120px;
+  background-color: #1c1a19;
+}
```

Comment:
- [ ] Please note that the `about me` section should be aligned with `flexbox` like mentioned in the requirements ,kindly fix it